source/core/src/main/com/csse3200/game/components/tasks/ItemProximityTask.java

```diff
@@ -95,15 +82,21 @@ public int getPriority() {
 
     @Override
     public void handleTargetMovedAway() {
-        if (this.itemOverlay != null) {
-                itemOverlay.dispose();
-                itemOverlay = null;
-            }
+        ServiceLocator.getEntityChatService().hideCurrentOverlay();
+    }
+
+    /**
+     * Checks if the player is near the item using the distance between the item and the player
+     * @return true if the player is within the proximity threshold, false otherwise.
+     */
+    private boolean isPlayerNearItem() {
```

Comment:
This method is unused and it's functionality is provided by ProximityTask

Reply:
Fixed